### PR TITLE
Update osv-scanner-pr.yml

### PR DESCRIPTION
**Pull request by hayleycd**
I was getting failing tests because it needs write access.

### CODE REVIEW

File: .github/workflows/osv-scanner-pr.yml [hunk 1]
```diff
@@ -25,5 +25,14 @@ on:
 permissions: read-all
 
 jobs:
-  scan-pr-attempt:
-    uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
+  scan-pr:
+    uses: "./.github/workflows/osv-scanner-reusable-pr.yml"
```
Comment (hayleycd) [inline .github/workflows/osv-scanner-pr.yml:29]:
```suggestion
    uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
```

File: .github/workflows/osv-scanner-pr.yml [hunk 2]
```diff
@@ -25,5 +25,14 @@ on:
...
 jobs:
-  scan-pr-attempt:
+  scan-pr:
     uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
+    with:
+      # Just scan the root directory and docs, since everything else is fixtures
+      scan-args: |-
+          --skip-git
+          ./
+          ./docs/
```
Comment (hayleycd) [inline .github/workflows/osv-scanner-pr.yml:35]:
```suggestion
```

File: .github/workflows/osv-scanner-pr.yml [hunk 3]
```diff
@@ -25,5 +25,14 @@ on:
...
 
 jobs:
-  scan-pr-attempt:
+  scan-pr:
     uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
+    with:
+      # Just scan the root directory and docs, since everything else is fixtures
+      scan-args: |-
+          --skip-git
+          ./
```
Comment (hayleycd) [inline .github/workflows/osv-scanner-pr.yml:34]:
```suggestion
```

File: .github/workflows/osv-scanner-pr.yml [hunk 4]
```diff
@@ -25,5 +25,14 @@ on:
 permissions: read-all
 
 jobs:
-  scan-pr-attempt:
+  scan-pr:
     uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
+    with:
+      # Just scan the root directory and docs, since everything else is fixtures
+      scan-args: |-
```
Comment (hayleycd) [inline .github/workflows/osv-scanner-pr.yml:32]:
```suggestion
```

File: .github/workflows/osv-scanner-pr.yml [hunk 5]
```diff
@@ -25,5 +25,14 @@ on:
 permissions: read-all
 
 jobs:
-  scan-pr-attempt:
+  scan-pr:
     uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
+    with:
+      # Just scan the root directory and docs, since everything else is fixtures
```
Comment (hayleycd) [inline .github/workflows/osv-scanner-pr.yml:31]:
```suggestion
```

File: .github/workflows/osv-scanner-pr.yml [hunk 6]
```diff
@@ -25,5 +25,9 @@ on:
 permissions: read-all
 
 jobs:
-  scan-pr-attempt:
+  scan-pr:
     uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
+    with:
```
Comment (hayleycd) [inline .github/workflows/osv-scanner-pr.yml:30]:
```suggestion
```

**Comment by hayleycd** [comment]
closing in favor of #1765

**Comment by another-rex** [comment]
Huh, for some reason I thought this was already merged.

**Pull request closed without merging**